Formula/libbi.rb

```diff
@@ -0,0 +1,156 @@
+class Libbi < Formula
+  desc "Bayesian state-space modelling on parallel computer hardware"
+  homepage "http://libbi.org"
+  head "https://github.com/libbi/LibBi.git"
+
+  stable do
+    url "https://github.com/libbi/LibBi/archive/1.3.0.tar.gz"
+    sha256 "0dd313dd71e72b2f16ca9074800fc2fa8bf585bec3b87a750ff27e467a9826d0"
+  end
+
+  option "without-test", "Disable build-time checking (not recommended)"
```

Comment:
Remove the option, and remove all tests below. We only perform build-time checking for security software.